vulnfeeds/cmd/combine-to-osv/main_test.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"encoding/json"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/google/osv/vulnfeeds/cves"
+)
+
+func loadTestData(cveName string) cves.CVEItem {
+	file, err := os.Open("../../test_data/nvdcve-1.1-test-data.json")
+	if err != nil {
+		log.Fatalf("Failed to load test data")
+	}
+	var nvdCves cves.NVDCVE
+	json.NewDecoder(file).Decode(&nvdCves)
+	for _, item := range nvdCves.CVEItems {
+		if item.CVE.CVEDataMeta.ID == cveName {
+			return item
+		}
+	}
+	log.Fatalf("test data doesn't contain specified CVE")
+	return cves.CVEItem{}
+}
+
+func sliceEqual[K comparable](a []K, b []K) bool {
+	if len(a) != len(b) {
+		return false
+	}
+	for i := 0; i < len(a); i++ {
+		if a[i] != b[i] {
+			return false
+		}
+	}
+	return true
+}
+
+func TestLoadParts(t *testing.T) {
+	allParts := loadParts("../../test_data/parts")
+
+	if len(allParts) != 10 {
+		t.Errorf("Expected 10 entries, got %d entries", len(allParts["Alpine"]))
+	}
+
+	hasCve := 0
+	for id, v := range allParts {
```

Comment:
Is this a good candidate for https://github.com/golang/go/wiki/TableDrivenTests?

Reply:
Yep, converted into a more table driven version now